src/main/resources/templates/mdBenefitsFlow/howThisWorks.html

```diff
@@ -24,10 +24,10 @@
         <th:block th:replace="~{fragments/honeycrisp/accordion :: accordion(
           buttonLabel=#{how-this-works.who-can},
           content=~{::who})}">
-            <th:block th:ref="who">
-             <p th:text="#{how-this-works.who-can-l1}"></p>
-              <p th:text="#{how-this-works.who-can-l2}"></p>
-            </th:block>
+          <th:block th:ref="who">
```

Comment:
I was going to edit something in this file and then changed my mind. But it did some auto formatting in the mean time.